src/app/Library/CrudPanel/Traits/ColumnsProtectedMethods.php

```diff
@@ -184,7 +184,7 @@ protected function hasDatabaseColumn($table, $name)
     {
         static $cache = [];
 
-        if ($this->driverIsMongoDb()) {
+        if (! $this->driverIsSql()) {
             return true;
         }
```

Comment:
I totally agree with this! If it's not something "we know", fallback to true. 
But ... is this a breaking change? 😣

Since some developer may be using other DB driver that's not SQL nor MongoDB? It was previously not falling to true...

I don't know what DB engines (other than mongo and sql) developers may be using, and probably if there are developer using other engines they would have reported issues related to those right!?
So I think we may not need to worry about this, and this method is not super crucial is it? 😬

So, once again, I'm just pointing a possible issue, please ignore if it's not relevant 😅

Reply:
Hello @promatik 

It would not be breaking change because in current code we only return true in case driver is mongo db, for all other drivers we run the database check (that will only work in sql databases like the ones in the list). So if people use other DB arch than MongoDB it would have failed already because we would had tried to run sql commands on it.

Reply:
Perfect 👌